app/Foundation/Exceptions/Displayers/ThrottleDisplayer.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Foundation\Exceptions\Displayers;
+
+use Exception;
+use GrahamCampbell\Exceptions\Displayers\DisplayerInterface;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpKernel\Exception\TooManyRequestsHttpException;
+
+class ThrottleDisplayer implements DisplayerInterface
```

Comment:
This class basically says, if throttling occurred on an auth route, go back to the login page with a translated rate limit flash message.